kotlin/Deepgram-Topic-Detection/README.md

```diff
@@ -0,0 +1,63 @@
+# 🧹 Deepgram Transcribe Audio
+
+A Cloud Function for transcribing audio using [Deepgram](https://deepgram.com/)
+
+_Example input:_
+
+```json
+{
+    "fileUrl":"https://static.deepgram.com/examples/interview_speech-analytics.wav"
+}
+```
+
+_Example output:_
+
+
+```json
+{ 
+    "deepgramData": {}, 
+    "success": true 
+}
+```
+
+_Example error output:_
+
+
+```json
+{
+    "success":false,
+    "message":"Error"
+}
+```
+
+## 📝 Environment Variables
+
+**DEEPGRAM_API_KEY** - Your Deepgram secret API key.
+Details are under link: [Deepgram_getting_started](https://developers.deepgram.com/documentation/getting-started/)
+
+## 🚀 Deployment
+
+1. Clone this repository, and enter this function folder:
+
+```bash
+$ git clone https://github.com/open-runtimes/examples.git && cd examples
+$ cd kotlin/Deepgram-Topic-Detection
```

Comment:
Would you please rename the folder to lowercase to match the folder name of the other examples?

Reply:
@JayantGoel001, this doesn't look like it's been addressed.

Reply:
Done